erd.md

```diff
@@ -0,0 +1,29 @@
+# Yelp Crow's Feet Notation
+## Entity Relationship Modeling
+
+### Diagram
+
+![Entity relationship model](diagram.png)
+
+### Description
+
+**Entities Chosen:**
+
+* Business
+* Customer
+* Orders
+* Reviews
+
+**As the model above shows:**
+* One `Business` can have many `Orders`, or they can have none if they're new or they make bad food
+* Many `Orders` can be created by one `Customer`, if they have the bread and want the butter
+* Many `Reviews` are created by exactly one `Customer` for each order, if the food is great... or horrible
+* Many `Orders` have many `Reviews`, so that the next `Customer` does't have to suffer
```

Comment:
This implies a M:M relationship between orders and reviews but your diagram shows a 1:M between the two. Do you mean _one_ order can have many reviews OR a review can belong to many orders?